docs/concepts/encryption.md

```diff
@@ -4,109 +4,68 @@ linkTitle: "Ondat Data Encryption"
-the Etcd and kubernetes secrets [can also be
-encrypted](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/),
-many organisations choose to use an external KMS provider.
-
-To address this from a Kubernetes limitations perspective and provide an
-agnostic solution, our encryption design allows the user to benefit from any
-Kubernetes KMS provider plugin to envelop the secrets into the KMS provider
-encryption scheme.
-
-Ondat allows customers to transparently integrate any supported KMS plugin
-with Ondat encryption key management using the standard Kubernetes API and
-Kubernetes KMS provider framework. The below figure provides an overview of the
-process.
+To address this from a Kubernetes limitations perspective and provide an agnostic solution, Ondat's encryption design allows the user to leverage any supported [Kubernetes KMS plugin](https://kubernetes.io/docs/tasks/administer-cluster/kms-provider/#implementing-a-kms-plugin) to envelop the secrets into a KMS provider encryption scheme.
+- Ondat enables end users to transparently integrate any supported KMS plugin with Ondat encryption key management using the standard Kubernetes API and Kubernetes KMS provider framework. The architecture diagram below provides a high level overview of the process.
 
 ![KMS Key Management](/images/docs/gui-v2/kms-key-management.png)
 
 1. The KMS plugin is deployed within the Kubernetes cluster.
-2. The KMS plugin is configured to act as a broker between the Kubernetes API
-   server and the KMS server API endpoint.
-3. At volume creation, Ondat will create a Kubernetes secret using
-   Kubernetes API calls
-4. The KMS plugin will handle the Kubernetes API Secret creation call and
-   interface to the KMS server instance.
-5. The KMS server will return the secret using its encryption envelop scheme.
-6. The KMS plugin will store the encrypted secret within the Kubernetes Etcd.
-
-Contact the [Ondat sales team](mailto:sales@storageos.com) for more
-information about the dedicated Ondat Vault KMS plugin integration
-offering.
+1. The KMS plugin is configured to act as a broker between the Kubernetes API server and the KMS server API endpoint.
+1. At volume creation, Ondat will create a Kubernetes secret using Kubernetes API calls
+1. The KMS plugin will handle the Kubernetes API Secret creation call and interface to the KMS server instance.
+1. The KMS server will return the secret using its encryption envelop scheme.
+1. The KMS plugin will store the encrypted secret within Kubernetes `etcd` backing store.
+
+> 💡 End users can also leverage [Trousseau](https://www.ondat.io/trousseau) with Ondat's volume encryption feature. Trousseau is an open source KMS plugin project that based on Kubernetes KMS provider design. The project allows users to store and access your secrets the Kubernetes native way with any external KMS. Trousseau's repository can be located on [GitHub](https://github.com/ondat/trousseau). 
```

Comment:
should we move it to the top of the page rather than the bottom? what do you think @hubvu ?

Reply:
Yes, I will go ahead and make the change.